### DISCUSSION TITLE
Opeb-WebUI has slowed down, what housekeeping \ maintenance is recommended.

### DISCUSSION THREAD

**Discussion by spammenotinoz** [open-webui/open-webui#7397] [General]
Hi team thankyou this project is working well, but recently there has been a noticeable speed reduction\delays;
- after login
- loading old chats (often will not load)
- cloning, renaming chats
Other than vacuuming the sqlite db, what housekeeping\maintenance is recommended?

Should I perhaps consider a migration to another DB such as postgres?  If so is there a migration guide?

WebUI.db is about 1.4 gb
Vector database is about 200mb

Cache folder is quite large, should this be pruned?

Thank-you, and really do apologies if this was in the docs, I have read \ searched before posting,

**Comment by tjbck** [comment]
Postgres is already supported, we do not have an official migration script but would love to see a contribution here!

Cache folders can also be pruned, however I'd recommend keeping it as is if possible. Hope that helps!

**Comment by spammenotinoz** [comment]
Thank-you for the quick response, I performed the DB vacuum and well it seems faster than day 1, so I won't need to migrate the DB.
Could be a good tip for others just for general housekeeping.

Any issues other than chat history, if prune to keep say 6 months of data in these folders?
/open-webui/data/cache/image/generations  (I no longer use this feature)
/open-webui/data/cache/audio (I've probably gone overboard testing this feature...)

Side note, I need to revisit my workflow, as I have many duplicates in /open-webui/data/uploads

**Comment by spammenotinoz** [comment]
For others;
- Vaccuuming the DB seemed to help a lot, I now do this weekly as part of my backup routine.
I also clean out old files from the following;
- /open-webui/data/cache/audio
- /open-webui/data/cache/transcripts
